app/src/components/market/common/market_scale/value_box.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import ReactTooltip from 'react-tooltip'
+import styled from 'styled-components'
+
+import { IconInfo } from '../../../common/tooltip/img/IconInfo'
+import { Circle } from '../../common/common_styled'
+import { SCALE_HEIGHT, VALUE_BOXES_MARGIN } from '../common_styled'
+
+const ValueBoxSegment = styled.div`
+  padding: 12px;
+  border: 1px solid ${props => props.theme.scale.box};
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 50%;
+  background: white;
+
+  &:nth-of-type(odd) {
+    border-top-right-radius: 0px;
+    border-bottom-right-radius: 0px;
+    border-top-left-radius: 4px;
+    border-bottom-left-radius: 4px;
+  }
+  &:nth-of-type(even) {
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
+    border-top-left-radius: 0px;
+    border-bottom-left-radius: 0px;
+    border-left: none;
+  }
+`
+
+const ValueBoxSingle = styled.div<{ xValue?: number }>`
+  padding: 12px;
+  border: 1px solid ${props => props.theme.scale.box};
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  ${props =>
+    props.xValue
+      ? props.xValue <= 0.885
+        ? `left: ${
+            props.xValue <= 0.115
+              ? `0`
+              : props.xValue <= 0.885
+              ? `${props.xValue * 100}%; transform: translateX(-50%);`
+              : ``
+          }`
+        : `right: 0;`
+      : ''}
+  background: white;
```

Comment:
should we maybe use value from theme here? and also closing column above it for the xValue...
![Screenshot 2021-02-12 at 12 36 09](https://user-images.githubusercontent.com/33226956/107763407-e3a38600-6d2e-11eb-8853-054b699625bb.png)